src/modules/curtainBuilder/components/builder/components/stepper/components/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Stepper from "@mui/material/Stepper";
+import Step from "@mui/material/Step";
+import StepLabel from "@mui/material/StepLabel";
+import DoneIcon from "@mui/icons-material/Done";
+import { StepIconProps } from "@mui/material/StepIcon";
+import { ICustomStepper } from "../interfaces";
+import style from "../styles/steps.module.scss";
+
+const CustomStepper: React.FC<ICustomStepper> = ({ activeStep, steps }) => {
+  let stepNum = 0;
+
+  const setCustomStepIcon = (props: StepIconProps) => {
+    const { completed, className } = props;
+
+    stepNum++;
+    return (
+      <div className={className}>
+        {completed ? (
+          <DoneIcon className="QontoStepIcon-completedIcon" />
+        ) : (
+          <span>{stepNum}</span>
+        )}
+      </div>
+    );
+  };
+
+  return (
```

Comment:
Function should have only 1 return point, this should tell you that Stepper supposed to be a separate component